welcome.css

```diff
@@ -75,12 +75,12 @@ html, body {
   color: olive;
 }
 
-.fa-bars::before {
+.burger-icon:before {
```

Comment:
`.burger-icon` is not good name...
I think it's a food hamburger icon.
If you are having trouble naming it, please leave a comment or a code for that role, and we will think about it together.

Reply:
<img width="1353" alt="Screenshot 2024-07-10 at 0 50 31 PM" src="https://github.com/scheme-dojo/scheme-dojo.github.io/assets/143992531/de0df8d0-b700-4fa2-bdf1-61867bf58e84">


the name of  "☰"  is hamburger menu icon,  it indicate  that it's a menu so I didn’t put other name.

Reply:
Thank you for sharing.
Naming is OK.


Please add comments even if it is an HTML file.
like that

```
<!-- Style when viewed on mobile-->
<i class="burger-icon"></i>

```

Reply:
okay, i well
Thank you